flagging_site/blueprints/reach_api.yml

```diff
@@ -0,0 +1,52 @@
+Endpoint returning json of the output model
+---
+tags:
+  - Reach Model API
+parameters:
+  - name: reach
+    description: type of reach model
+    in: query
+    type: int
+    enum: [2, 3, 4, 5]
+    required: false
+    default:
+  - name: hour
+    description: type of reach model
+    in: query
+    type: int
+    enum: [1, 2, 3, 6, 12, 24, 36, 48]
+    required: false
+    default:
+responses:
+  200:
+    description: Dictionary-like json of the output model
+    schema:
+      id: model_api
+      type: object
+      properties:
+        version:
+          description: the year schema is made
+          type: string
+        time_returned:
+          description: time data is returned
+          type: string
+        models:
+          description: the year schema is made
+          type: object
+          properties:
+            model_num:
+              description: type of reach model
+              type: object
+              properties:
+                time:
+                  description: time of data
+                  type: string
+                log_odds:
+                  description: logistical odds
+                  type: number
+                probability:
+                  description: probability
+                  type: number
+                safe:
+                  description: whether or not river is safe
+                  type: boolean
```

Comment:
Try this instead:

```
                time:
                  description: time of data
                  type: array
                  items:
                    type: string
                log_odds:
                  description: log odds output of the model
                  type: array
                  items:
                    type: number
                probability:
                  description: probability output of the model
                  type: array
                  items:
                    type: number
                safe:
                  description: whether or not river is safe
                  type: array
                  items:
                    type: boolean
```

This is technically what the schema is-- e.g. we're returning an array of strings, not just a string.

Also, it's just log odds, not logistical odds. This is what it's in reference to: https://en.wikipedia.org/wiki/Logit The log odds is basically the inverse of the sigmoid function (which converts log odds into a probability).

Reply:
Thanks! I want to demo it just to get people's feedback because I am not too sure how we should present the documentation especially the schema.

Reply:
I think if you make the update I suggest above it'll be fine.

I don't want to pretend as if there aren't any design choices that are up to opinion, but really a schema's specification is something you've either described, or you haven't. And right now, it's not being described perfectly because you're saying the output is a string or a bool etc. which is false. The outputs are _arrays_ of strings and bools.

Reply:
Pushes the changes. Changed the types to arrays.